tests/integration/relations/test_db.py

```diff
@@ -213,6 +214,7 @@ async def test_relation_with_indico(ops_test: OpsTest):
     )
 
 
+@pytest.mark.unstable
 async def test_relation_with_openldap(ops_test: OpsTest):
```

Comment:
Test fails consistently, created [DPE-2685](https://warthogs.atlassian.net/browse/DPE-2685) to investigate